utils/releaser/src/ReleaseWorker/Release/CreateAndPushGitTagsExceptProjectBaseReleaseWorker.php

```diff
@@ -5,26 +5,53 @@
 namespace Shopsys\Releaser\ReleaseWorker\Release;
 
 use PharIo\Version\Version;
+use Shopsys\Releaser\Packagist\PackageProvider;
 use Shopsys\Releaser\ReleaseWorker\AbstractShopsysReleaseWorker;
 use Shopsys\Releaser\Stage;
 
 final class CreateAndPushGitTagsExceptProjectBaseReleaseWorker extends AbstractShopsysReleaseWorker
 {
-    private const PACKAGES_WITHOUT_MONOREPO_AND_PROJECT_BASE = [
-        'http-smoke-testing',
-        'coding-standards',
-        'read-model',
-        'product-feed-google',
-        'product-feed-zbozi',
-        'product-feed-heureka-delivery',
-        'product-feed-heureka',
-        'plugin-interface',
-        'framework',
-        'form-types-bundle',
-        'migrations',
-        'google-cloud-bundle',
+    /**
+     * Packages that are not on Packagist, so unable to found by API, but also running on Travis
+     * @var string[]
+     */
+    private const EXTRA_PACKAGES = [];
+
+    /**
+     * Packages that are not tested on Travis - old packages or forks
+     * @var string[]
+     */
+    private const EXCLUDED_PACKAGES = [
+        // excluded from the initial tagging as there needs to be another commit with composer.lock and package-lock.json
+        // @see https://github.com/shopsys/shopsys/pull/1264
+        'shopsys/shopsys',
+        'shopsys/project-base',
+        'shopsys/product-feed-interface',
+        'shopsys/phpstorm-inspect',
+        // forks
```

Comment:
from here down, it is DRY violation again, however, it might be justified as the list may differ from the one used in `TravisStatusReporter`. 

But maybe it would make sense to create a list of by default excluded packages in `PackageProvider` and then use the default list here and in `TravisStatusReporter` a let developers to extend the array somehow (like here we need to add `shopsys/shopsys` and `shopsys/project-base` to the array :thinking:

Reply:
I personally think that it should not be in one basic array as (as you said) the arrays may differ in the future

Reply:
ok :)